DesktopUI2/DesktopUI2/Views/Windows/Dialogs/MissingIncomingTypesDialog.xaml

```diff
@@ -14,13 +14,20 @@
 
     <TextBlock
       Grid.Row="1"
-      Margin="15"
+      Margin="15,15,15,5"
+      Foreground="Gray"
+      Text="Some incoming types are missing, would you like to import or map them?"
```

Comment:
This doesn't necessarily require action, but I wonder if any of our users would not know what it means to "map" types